Webinar details pagina sidney

Sidopjescherm opened the pull request:
Hier is de basis opzet van mijn webinars detail pagina. Ik wil dit eigenlijk in kleine stapjes doen, want dit moet volgens de code conventies. Daarom hier als eerst het begin. Zo kan ik straks ook de fonts gebruiken in de pagina, want ik heb die update van de stylesheet nog niet. 

Zouden jullie willen kijken naar: 

De clamps op de teksten en de video
De one column layout op mobile
Wat jullie eraan zouden veranderen
Hoe je de ongemakkelijke gap weghaald waar de tekst plaats maakt

<img width="523" height="69" alt="Screenshot 2025-10-14 133101" src="https://github.com/user-attachments/assets/359c6cfe-bed8-45c5-92da-2928633fd152" />
<img width="525" height="311" alt="Screenshot 2025-10-14 133108" src="https://github.com/user-attachments/assets/2fe2b705-6114-4a34-bd8e-ed5b52b91450" />
<img width="522" height="413" alt="Screenshot 2025-10-14 133114" src="https://github.com/user-attachments/assets/0106e7a8-69e0-4839-a680-edea63e61bf3" />

Sidopjescherm committed:
#270 Update

Hiermee heb ik de versies van mijn modules een update gegeven

Sidopjescherm committed:
#270 Params webinar

Omdat ik niet via mijn computer van branch kan veranderen moet ik de pagina opnieuw maken. Hierbij alleen de webinars setup die ik heb gemaakt. Het laad nu alleen de webinars in, maar nog niet de speakers zelf en de categorieën.

Sidopjescherm committed:
#270 Basic opzet

HIerbij de basic opzet van de HTML pagina met een aantal ingeladen dynamische data. Er is nog geen styling toegepast en de footer en de header zijn ook nog niet gemaakt.

Sidopjescherm committed:
#270 Basic styling toegevoegd

Ik heb een basic styling toegevoegd die nog wat verfijning nodig heeft

Sidopjescherm committed:
#270 Error throw en styling

Hier is een kleine commit over de styling en de error throw die ik maak

Sidopjescherm committed:
#270 Basic styling af

Hier is de basic styling af van de details webinar pagina

nadiachaja reviewed (changes requested):
- Ik zou even kijken hoe ik de kruimelpad heb gemakt nu ziet dat er anders uit dan in het Figma bestand. 
- Als je voor de font-size clamp gebruikt zet dat dan als je de css hebt in een custom propertie. 
- Ik zou meer padding om de buttons en linkjes doen. 
- Wat is de hr?
- De html ziet er goed uit.
- Welke gap bedoel je?

Sidopjescherm committed:
#276 Veranderingen review

@nadiachaja ik heb de veranderingen toegepast die waren vernoemd in de review. Ik heb mijn elementen die te diep genest waren eruit gehaald zodat oude browsers dit nu ook ondersteunen. Ook heb ik het tekentje veranderd zoals het in Figma staat en in jouw bruidkruimel pad op de webinars overzicht pagina

nadiachaja reviewed (approved):
ziet er goed uit

Sidopjescherm committed:
Merge branch 'dev' into webinar_Details_Pagina_Sidney
